src/app.ts

```diff
@@ -52,7 +52,9 @@ jwtVerifier
     console.error(`Failed to hydrate JWT verifier : ${e}`);
   })
   .then(() => {
-    app.listen(port, () => {
+    const server = app.listen(port, () => {
       console.log(`listening on port: ${port}!`);
     });
+
+    server.setTimeout(5000);
```

Comment:
I would maybe consider bumping this to 10 just to give ourselves some more breathing room in high load scenarios, but I don't know the system as well so I trust your gut here.